modules/web-console/web-console-server/src/main/java/org/apache/ignite/console/common/Utils.java

```diff
@@ -129,4 +130,12 @@ public static String currentRequestOrigin() {
             .build()
             .toString();
     }
+
+    /**
+     * @param rows Number of rows.
+     * @return Object with number of affected rows.
+     */
+    public static RowsAffected rowsAffected(int rows) {
```

Comment:
Why we need this. At frontend code we don't use it.

Reply:
Removed RowsAffected  class.